packages/gatsby-cli/src/index.ts

```diff
@@ -36,7 +34,7 @@ process.on(`unhandledRejection`, reason => {
   // across versions and crash
 
   // reason can be anything, it can be a message, an object, ANYTHING!
-  // we convert it to an error object so we don't crash on structured error validation
+  // we convtert it o an error object so we don't crash on structured error validation
   if (!(reason instanceof Error)) {
     reason = new Error(util.format(reason))
   }
```

Comment:
I had to convert a second package to ensure the check-ts script could properly handle our monorepo. Good news, it does now!